src/cloud-formation/deploy-dev.yaml

```diff
@@ -1,20 +1,24 @@
 ---
 parameters:
-  BrokerImageVersion: '0.3.0'
+  BrokerImageVersion: '0.7.0'
   Environment: 'dev'
-  MaybeDeployAlb: 'false'
-  MaybeDeployAlbDnsRecord: 'false'
-  MaybeDeployBastionHost: 'false'
-  MaybeDeployBroker: 'false'
-  MaybeDeployContainers: 'false'
-  MaybeDeployDb: 'false'
-  MaybeDeployPostgrest: 'false'
-  MaybeDeployProcessor: 'false'
-  MaybeDeployRouteTables: 'false'
-  MaybeDeployStack: 'false'
-  MaybeDeployVpc: 'false'
+  MaybeDeployAlb: 'true'
+  MaybeDeployAlbDnsRecord: 'true'
+  MaybeDeployBastionHost: 'true'
+  MaybeDeployBroker: 'true'
+  MaybeDeployContainers: 'true'
+  MaybeDeployDb: 'true'
+  MaybeDeployNlb: 'true'
+  MaybeDeployNlbVpcLink: 'true'
+  MaybeDeployPostgrest: 'true'
+  MaybeDeployProcessor: 'true'
+  MaybeDeployRestApi: 'true'
+  MaybeDeployRestApiDnsRecord: 'true'
+  MaybeDeployRouteTables: 'true'
+  MaybeDeployStack: 'true'
+  MaybeDeployVpc: 'true'
   Network: 'testnet'
-  ProcessorImageVersion: '0.4.0'
+  ProcessorImageVersion: '0.5.0'
```

Comment:
Are these versions actually changing or are you bumping it to trigger a re-build? Just curious